.buildkite/ml.rayci.yml

```diff
@@ -213,6 +213,20 @@ steps:
         --only-tags ptl_v2
     depends_on: [ "mllightning2gpubuild", "forge" ]
 
+  - label: ":train: ml: :windows: tests"
+    tags:
+      - train
+    job_env: WINDOWS
+    instance_type: windows
+    commands:
+      - bash ci/ray_ci/windows/install_tools.sh
+      - docker tag rayproject/buildenv:windows $${RAYCI_WORK_REPO}:$${RAYCI_BUILD_ID}-mlbuild
+      - bazel run //ci/ray_ci:test_in_docker -- //python/ray/train:test_windows ml
+        --operating-system windows
+        --test-env=CI="1"
+        --test-env=RAY_CI_POST_WHEEL_TESTS="1"
+        --test-env=USERPROFILE="$${USERPROFILE}"
```

Comment:
why does the test need `USERPROFILE`?

also, this will be sending build launcher's `USERPROFILE` into the container? does not seem quite right to me.

Reply:
I just copy this from `windows_ci.sh test_train_windows` (https://github.com/ray-project/ray/blob/master/ci/ci.sh#L283)

Reply:
that one was ok because it is not wrapped inside a docker. this one will be expanded on the shell here (when running `bazel run test_in_docker` rather than inside the docker.

I think you can just send in `--test-env=USERPROFILE`

or just drop this line maybe.